src/data/polynomial/degree/trailing_degree.lean

```diff
@@ -169,10 +180,7 @@ end
 @[simp] lemma coeff_nat_trailing_degree_pred_eq_zero {p : polynomial R} {hp : (0 : with_top ℕ) < nat_trailing_degree p} : p.coeff (p.nat_trailing_degree - 1) = 0 :=
```

Comment:
GitHub won't let me make a suggestion here, but this can be turned into term mode

```lean
@[simp] lemma coeff_nat_trailing_degree_pred_eq_zero {p : polynomial R} {hp : (0 : with_top ℕ) < nat_trailing_degree p} : p.coeff (p.nat_trailing_degree - 1) = 0 :=
coeff_eq_zero_of_lt_nat_trailing_degree $ nat.sub_lt
  ((with_top.zero_lt_coe (nat_trailing_degree p)).mp hp) nat.one_pos
```

Reply:
Changed the proof to term mode.
Thanks for the suggestion!

Reply:
Changed the proof to term mode.

Thanks for the suggestion!